### PR TITLE
docs: various corrections to components docs

### PR DESCRIPTION
**Pull request by vvtimofeev**
I added various corrections, to documentation.
Corrections proposed by the redactors team. 
Please, review them.

### CODE REVIEW

**Comment by korvin89** [comment]
U have linting errors https://github.com/gravity-ui/uikit/actions/runs/6337425662/job/17212493747?pr=1030#step:5:29

**Review by korvin89 — APPROVED** [review]

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/ArrowToggle/README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 <!--/GITHUB_BLOCK-->
 
-`ArrowToggle` is a component that displaying chevron icon. It can rotate in 4 directions. It could be used for displaying dropdown lists or cut components, etc.
+`ArrowToggle` is a component for displaying the chevron icon. It be rotated to point in four directions, and can be usef to display dropdown lists, cut components, etc.
```
Comment (Marginy605) [inline src/components/ArrowToggle/README.md:7]: `It be rotated to point` sounds a little bit weird

File: src/components/ArrowToggle/README.md [hunk 2]
```diff
@@ -115,8 +115,8 @@ return (
...
 
-| Name      | Description                 |   Type   |  Default   |
-| :-------- | :-------------------------- | :------: | :--------: |
-| className | HTML `class` attribute      | `string` |            |
-| direction | Sets arrowToggle directions | `string` | `"bottom"` |
-| size      | Size arrowToggle in px      | `number` |    `16`    |
+| Name      | Description                          |   Type   |  Default   |
+| :-------- | :----------------------------------- | :------: | :--------: |
+| className | HTML `class` attribute               | `string` |            |
+| direction | Used to set direction of arrowToggle | `string` | `"bottom"` |
```
Comment (Marginy605) [inline src/components/ArrowToggle/README.md:121]: Looks like it needs an article before `direction`

**Review by Marginy605 — CHANGES_REQUESTED** [review]

File: src/components/ArrowToggle/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vvtimofeev) [reply src/components/ArrowToggle/README.md]: Thank you. Fixed

File: src/components/Link/README.md [hunk 3]
```diff
@@ -134,7 +134,7 @@ LANDING_BLOCK-->
 
 ## Usage
 
-Сan be used as an independent text element, or as part of the text
+A 'Link' can be used both as an independent text element and as part of the text:
```
[the remark this answers is not in the document]
Comment (vvtimofeev) [reply src/components/Link/README.md]: Fixed

File: src/components/ArrowToggle/README.md (hunk 1 above)
Comment (vvtimofeev) [reply src/components/ArrowToggle/README.md]: Thank you. Reverted it to the old variant.

File: src/components/ArrowToggle/README.md (hunk 2 above)
Comment (vvtimofeev) [reply src/components/ArrowToggle/README.md]: Added it!

**Review by Marginy605 — APPROVED** [review]

**Review by Lunory — APPROVED** [review]

**Review by jhoncool — APPROVED** [review]

**Review by amje — APPROVED** [review]

**Pull request merged**